File: Solutions/1323. Maximum 69 Number/1323. Maximum 69 Number.cpp
class Solution {
public:
    int maximum69Number (int num) {
        int maxdigit=0,currdigit=1;
		int n=num;
		while(n>0) {
			if(n%10==6)maxdigit=currdigit;
			currdigit*=10;
			n/=10;
		}
		return num+3*maxdigit;
    }
};

File: Solutions/1323. Maximum 69 Number/1323. Maximum 69 Number.go
func maximum69Number (num int) int {
    maxdigit,currdigit:=0,1
    n:=num
    for n>0{
        if n%10==6{
            maxdigit=currdigit
        }
        currdigit*=10
        n/=10       
    }
    return num+3*maxdigit
}